org.ghidra_sre.Ghidra.json

```diff
@@ -48,13 +48,13 @@
             "name": "ghidra",
             "buildsystem": "simple",
             "build-commands": [
-                "for i in docs Extensions Ghidra GPL licenses server support ; do install -d $i /app/lib/ghidra/ ; cp -r $i/ /app/lib/ghidra/$i/ ; done",
```

Comment:
Explain why this is a necessary change in the commit message.

Reply:
Ok

Reply:
Done: 4002738b9491dac35ea1bda4d7ddd5859c206d03